### ISSUE TITLE
Rule Proposal: jsx-equals-spacing

### ISSUE THREAD

**Issue by peet** [yannickcr/eslint-plugin-react#394]
Simply a rule to enforce or disallow spaces around the = in JSX elements

```js
"jsx-equals-spacing": [2, "always" || "never"]
```

Warning when set to "never":
```js
<Hello name = {firstname} />;
```

Warning when set to "always":
```js
<Hello name={firstname} />;
```

Defaults to "never" to match Facebook examples

**Comment by peet** [comment]
Thanks!
